go/protocol/chat1/extras.go

```diff
@@ -68,14 +70,18 @@ const DbShortFormLen = 10
 
 // DbShortForm should only be used when interacting with the database, and should
 // never leave Gregor
-func (cid ConversationID) DbShortForm() []byte {
+func (cid ConversationID) DbShortForm() ConvIDShort {
 	return cid[:DbShortFormLen]
 }
 
 func (cid ConversationID) DbShortFormString() string {
 	return hex.EncodeToString(cid.DbShortForm())
 }
 
+func DbShortFormToString(cid ConvIDShort) string {
```

Comment:
Can we just make this `.String()` on `ConvIDShort`?

Reply:
We can't since it's an alias of `[]bytes` we can't override `String`

`invalid receiver type []byte ([]byte is an unnamed type)`